src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftBuilder.java

```diff
@@ -92,7 +93,8 @@ public DescriptorImpl getDescriptor() {
      */
     @Extension // This indicates to Jenkins that this is an implementation of an extension point.
     public static final class DescriptorImpl extends BuildStepDescriptor<Builder> {
-    	private long wait = GlobalConfig.BUILD_WAIT;
+    	private String wait = GlobalConfig.getBuildWait();
+        private String waitUnit;
```

Comment:
After discussion with Gabe, I've converted all the indentation in the files I worked in to tabs.